impeller/typographer/typographer_unittests.cc

```diff
@@ -244,40 +263,6 @@ TEST_P(TypographerTest, GlyphAtlasTextureIsRecycledIfUnchanged) {
   ASSERT_EQ(old_packer, new_packer);
 }
 
-TEST_P(TypographerTest, GlyphAtlasTextureIsRecreatedIfTypeChanges) {
```

Comment:
This test case is deleted because we never need to change the type of an atlas at runtime, we always create distinct alpha and color atlases